Documentation/RenderingDocs/Index.rst

```diff
@@ -10,97 +10,116 @@
-or :ref:`ask for help <get-help-on-writing-docs>`.
+#.  Clone the repository containing the documentation.
 
+#.  Navigate to the directory, which contains the :file:`composer.json`
 
-.. rst-class:: bignums-xxl
+#.  Check if there is documentation to be rendered:
 
-    #.  Install Docker: https://docs.docker.com/install/
+    A folder called :file:`Documentation` containing at least the files
+    :file:`Index.rst` and :file:`guides.xml`.
 
-    #.  Preparations
+#.  Choose your prefered method of rendering (See bellow):
 
-        The docker image is not listed on Docker Hub (hub.docker.com) anymore, therefore some preparations
-        need to be done once:
+Make sure that `Docker <https://www.docker.com/>`__ is installed on your system.
 
-        ..  code-block:: bash
+.. tabs::
 
-            # pull 'latest' version from GitHub container repository
-            docker pull ghcr.io/t3docs/render-documentation
+    ..  group-tab:: Linux
 
-            # The "real" tag is independent of the container repository,
-            # so let's just create that extra "real" and "generic" tag
-            docker tag ghcr.io/t3docs/render-documentation t3docs/render-documentation
+        ..  code-block:: bash
 
-            # verify it worked
-            docker run --rm t3docs/render-documentation --version
+            mkdir -p Documentation-GENERATED-temp
+            docker run --rm --pull always -v $(pwd):/project -it ghcr.io/typo3-documentation/render-guides:latest --config=Documentation
+            xdg-open "Documentation-GENERATED-temp/Index.html"
 
-    #.  Make 'dockrun_t3rd' available in current terminal
+    .. group-tab:: MacOS
 
-        .. code-block:: bash
+        ..  code-block:: bash
 
-            eval "$(docker run --rm t3docs/render-documentation show-shell-commands)"
+            mkdir -p Documentation-GENERATED-temp
+            docker run --rm --pull always -v $(pwd):/project -it ghcr.io/typo3-documentation/render-guides:latest --config=Documentation
+            open "Documentation-GENERATED-temp/Index.html"
 
-        This will run the Docker container and print out shell code. Once "eval-uated" 
-        the code defines a helper function named `dockrun_t3rd`. This function relieves 
-        you of the work of setting the volumes and rights correctly when running the 
-        container and ensures a folder for the rendering results is provided. Executing 
-        the code with `eval` defines the function for your current terminal. Don't forget
-        the quotes around everything that goes into eval. Either define the function each
-        time you open a new terminal window or add the line to the startup file of your
-        shell like `~/.bashrc` or `.zshrc`.
+    ..  group-tab:: Windows
 
-    #.  Run dockrun_t3rd
+        ..  code-block:: powershell
 
-        .. code-block:: bash
+            start "Documentation-GENERATED-temp/Index.html"
```

Comment:
What does "start"?

Reply:
It seems like on the mac it opens a browser and displays the file. The line was there before, I only adjusted the path. But I cannot verify it as I don't have a mac. Maybe @garvinhicking can help here

Reply:
Ah sorry, lost context. It is a powershell command which opens the document in a browser

Reply:
But then it should be moved below the rendering command. But there is still the "open" command.